tests/test_examples.py

```diff
@@ -500,7 +500,7 @@ def test_simsopt_QH_comparison():
     # Cancel factor of 1/2 in desc objective which is not present in simsopt:
     aspect_weight = np.sqrt(2)
     # Also scale QS weight so objective is approximately independent of grid resolution:
```

Comment:
why this factor, if it is not actually something that is related to the grid anymore?

Reply:
there's still a difference between us and simsopt, due to our theta/zeta being [0,2pi] and VMEC u,v being [0,1], and an extra factor of 1/2, but I realize I forgot a square root so ill add that.